src/drivers/dw/dma.c

```diff
@@ -557,9 +554,9 @@ static int dw_dma_set_config(struct dma_chan_data *channel,
 		if (dw_chan->lli)
 			rfree(dw_chan->lli);
 
-		dw_chan->lli = rballoc_align(0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
-					sizeof(struct dw_lli) * channel->desc_count,
-					PLATFORM_DCACHE_ALIGN);
+		dw_chan->lli = rzalloc(SOF_MEM_ZONE_RUNTIME_SHARED, 0,
+				       SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
+				       sizeof(struct dw_lli) * channel->desc_count);
```

Comment:
why does it have to be zeroed? It wasn't zeroed before. If that was causing problems because we expect zeroes somewhere, then that should be fixed explicitly! Otherwise `rmalloc()` should do just fine.

Reply:
So you don't need alignment any longer? That seems very odd for hardware descriptors?

Reply:
> So you don't need alignment any longer? That seems very odd for hardware descriptors?

@plbossart the rmalloc() internal guarantees the return pointer is PLATFORM_DCACHE_ALIGN aligned, please see the source here:
https://github.com/thesofproject/sof/blob/8fc5247289db0093bbb9de369c1dcb7e0d19a109/src/lib/alloc.c#L718